test/lit.cfg

```diff
@@ -139,6 +139,7 @@ class SwiftTest(lit.formats.ShTest, object):
             self.coverage_mode = None
         else:
             self.coverage_mode = coverage_mode
+        self.skip_merge_next_test = False
```

Comment:
Could you invert the semantics of this option to be `merge_next_test`?  Double-negations are harder to reason about.